[origin] OpenShift 3.10 updates

brenton opened the pull request:
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?

bmr-cymru commented:
```
$ pycodestyle ...
  '\n\n'
sos/plugins/origin.py:62:80: E501 line too long (82 > 79 characters)
sos/plugins/origin.py:131:13: E303 too many blank lines (2)
The command "pep8 sos" exited with 1.
```

bmr-cymru commented on sos/plugins/origin.py:
```diff
@@ -107,9 +118,16 @@ def setup(self):
+            if self.is_static_pod_compatible():
+                self.add_copy_spec([
+                    os.path.join(self.static_pod_dir, "*.yaml"),
+                ])
```
No need to use a list for a single item - `add_copy_spec()` accepts either a string, or a list of strings. The list is only needed if there is > 1 item (and this often helps to avoid the need for line breaks).

bmr-cymru commented on sos/plugins/origin.py:
```diff
@@ -155,15 +173,14 @@ def setup(self):
                 os.path.join(self.node_base_dir, "*.crt"),
             ])
 
-            node_kubecfg = self.get_node_kubecfg()
```
If you're deleting the only call to this method please also delete the method definition.

brenton commented:
I think I addressed most of the comments (however I just now noticed your commend about moving the member variables.  noted, I'll update that first thing in the morning).  I check and pycodestyle and pep8 runs were clean.

I added support for collecting logs from the master in 3.10 as well.

bmr-cymru commented:
Looks good to me - thanks. The variables are now defined closer to where they are used & there's no need for the extra `self.` qualifier.

Good catch on the duped log, I'd missed that.

brenton commented:
OK, I'll make a point to test this on an older version of OCP to just make sure it's also OK there.  If everything looks good I'll squash this down and let you know.

brenton commented:
@bmr-cymru,  FYI, my testing passed on older OCP versions as well.  I think this is ready from my perspective.  I squashed the PR down.

bmr-cymru commented:
:+1: thanks!

bmr-cymru commented:
Just time, but it's at the head of the queue now, so it should be in both 3.5.1 (this week), and 3.6 (another month or so).
